Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.3.7'
+ruby '2.5.1'
```

Comment:
only comment is wondering if there's a higher patch version for ruby. I think 2.5.1 was the latest stable when we made the ticket. Seems like 2.6 is a little too new to be supported, but maybe if there's a higher 2.5.x we could use that version

Reply:
nevermind, looks like 2.5.1 is the latest

Reply:
@akegan there's a 2.5.3

Reply:
https://www.ruby-lang.org/en/news/2018/10/18/ruby-2-5-3-released/

Reply:
I'm using 2.5.3 locally for couple project and it's fine. I upgraded to 2.5.1 as it was specified in the task.

Reply:
cool, well then @software-project go ahead and upgrade to 2.5.3 in this PR and check that everything still works. i believe 2.5.1 was specified in the task because that was the latest 2.5.x at the time

Reply:
wait, it looks like semaphore doesn't have 2.5.3 
![image](https://user-images.githubusercontent.com/11825994/53451788-7c228d80-39d4-11e9-9666-dce95172ba5d.png)

Reply:
oh that's a bit annoying. we can still install it ourselves in our setup steps though

Reply:
hmm rbenv doesn't seem to offer 2.5.3 for some reason, that would have been my plan for installing manually on semaphore, i'm going to look into that a bit to see if there is some reason why

Reply:
hmmm seems like heroku doesnt like 2.5.1. Deploy is successful but app is down

Reply:
may need to upgrade the buildpack

Reply:
ok looks like i just had to update my ruby-build, i now see 2.5.3 available through rbenv. we'll have to check that semaphore's ruby-build is also up-to-date

Reply:
Ok I see why

Reply:
haha i think we all commented at the exact same time

Reply:
@software-project we can use the instructions at the bottom of these [semaphore docs about ruby versions](https://semaphoreci.com/docs/languages/ruby/ruby-support-on-semaphore.html) to add installation of ruby 2.5.3 to our semaphore set up